pact/Pact/Core/Gas/TableGasModel.hs

```diff
@@ -157,6 +156,35 @@ intShiftCost _ !rop
   | rop > 0 = MilliGas $ fromIntegral $ rop `quot` 1000
   | otherwise = MilliGas 0
 
+intPowCost :: Integer -> Integer -> MilliGas
+intPowCost !base !power
```

Comment:
The gas for this seems incredibly complicated, and I've yet to test this locally.

What do you think about an approach such as [this one](https://github.com/kadena-io/pact/blob/master/src/Pact/Native/Ops.hs#L141)? It essentially simply charges on the actual operations that occur during the algorithm for integer exponentiation, which is simply a series of multiplications.

Reply:
I think we've resolved this offline, marking as such.